src/private/CABasicAnimation+MotionAnimator.h

```diff
@@ -24,7 +24,13 @@
 FOUNDATION_EXPORT
 CABasicAnimation *MDMAnimationFromTiming(MDMMotionTiming timing, CGFloat timeScaleFactor);
 
-// Attemps to configure the provided animation to be additive.
-// Not all animation value types are supported. If an animation's value type was not supported,
-// the animation will not be modified.
-FOUNDATION_EXPORT void MDMMakeAnimationAdditive(CABasicAnimation *animation);
+// Attempts to configure the provided animation to be additive and, if the animation is a spring
+// animation, will extract the initial velocity from the timing and apply it to the animation.
+//
+// Not all animation value types support being additive. If an animation's value type was not
+// supported, the animation's values will not be modified.
+//
+// This method assumes that the animation's from and to value do not match.
+FOUNDATION_EXPORT void MDMConfigureAnimation(CABasicAnimation *animation,
```

Comment:
return BOOL on success?

Reply:
It's unclear what success would mean here. If it means "some modification was made" then I'm not certain what value the caller will get out of knowing that information. Did you have a use case in mind?

Reply:
My understanding is that they wouldn't be calling  this modifier unless they expected the animation to become additive. If it cannot be made additive, they may take some other action. Maybe they won't animate, maybe they'll use a less-preferred solution?